00 Meta/Vault To Do.md

```diff
@@ -3,6 +3,10 @@
 ## Organization
 
 - [ ] Write MOCs or similar that encourage exploration, the current ones are awful (sorry!)
+- [ ] [[Introduce Use Cases]]
+	- [ ] Daily Notes
+	- [ ] Weekly Notes
+	- [ ] A Person page (last contact, birthday, ...)
```

Comment:
It might be helpful to also link to Zektor's template repository (don't have the link on me right now)

Reply:
I don't know that one. Why would it helpful here, to use it as a inspiration when building the corresponding use case?